### PR TITLE
Fronius: Update to latest interface requirements and add providedInterfaces 

### PR DESCRIPTION
**Pull request by t-mon**
…or the datalogger

nymea-plugins pull request checklist:

- [x] Make sure the pull request's title is of format "Plugin name: Add support for xyz" or "New plugin: Plugin name"

- [x] Did you test the changes on hardware, if not (e.g. absence of required hardware), please mention a person to confirm it has been tested.

- [x] Did you update the plugin's README.md accordingly?

- [x] Did you update translations (`cd builddir && make lupdate`)?

- [x] If you added a new plugin, should it be added to nymea-plugins-all or nymea-plugins-maker?

### CODE REVIEW

**Review by mzanetti — APPROVED** [review]

File: fronius/froniusstorage.cpp [hunk 1]
```diff
@@ -126,6 +126,6 @@ void FroniusStorage::updateActivityInfo(const QByteArray &data)
     QVariantMap dataMap = jsonDoc.toVariant().toMap().value("Body").toMap().value("Data").toMap();
     float charge_akku = dataMap.value("Site").toMap().value("P_Akku").toFloat();
     pluginThing()->setStateValue(storageCurrentPowerStateTypeId, charge_akku);
-    pluginThing()->setStateValue(storageChargingStateTypeId, charge_akku < 0);
-    pluginThing()->setStateValue(storageDischargingStateTypeId, charge_akku > 0);
```
Comment (mzanetti) [inline fronius/froniusstorage.cpp:126]: the interface specifies a single state for this.

**Review by mzanetti — CHANGES_REQUESTED** [review]

File: fronius/froniusstorage.cpp (hunk 1 above)
Comment (mzanetti) [reply fronius/froniusstorage.cpp:126]:
```
        {
            "name": "chargingState",
            "type": "QString",
            "allowdValues": ["idle", "charging", "discharging"],
            "optional": true
        }
```

Comment (t-mon) [reply fronius/froniusstorage.cpp:126]: fixed

File: fronius/integrationpluginfronius.json [hunk 2]
```diff
@@ -384,7 +384,7 @@
                     "name": "storage",
                     "displayName": "Fronius solar storage",
                     "createMethods": ["auto"],
-                    "interfaces": [ "energystorage", "connectable" ],
+                    "interfaces": [ "energystorage"],
```
Comment (mzanetti) [inline fronius/integrationpluginfronius.json:387]: accidental?

**Review by mzanetti — APPROVED** [review]

**Pull request merged**